assignment.js

```diff
@@ -111,7 +133,47 @@ Use the following test cases to confirm your program meets the success criteria
 
 console.log('Problem 3:')
 
-// Add your code below this line
+
+//Player B Wins
+let playerChoiceA = "rock"
+let playerChoiceB = "rock"
+
+if(playerChoiceA === "rock" && playerChoiceB === "paper") {
+    console.log("The winner is Player B")
+}
+if(playerChoiceA === "paper" && playerChoiceB === "scissors") {
+    console.log("The winner is Player B")
+}
+if(playerChoiceA === "scissors" && playerChoiceB === "rock") {
+    console.log("The winner is Player B")
+}
+
+//Player A Wins
+else if(playerChoiceB === "rock" && playerChoiceA === "paper") {
+  console.log("The winner is Player A")
+}
+else if(playerChoiceB === "paper" && playerChoiceA === "scissors") {
+  console.log("The winner is Player A")
+}
+else if(playerChoiceB === "scissors" && playerChoiceA === "rock") {
+  console.log("The winner is Player A")
+}
+
+//It's a tie
+else if(playerChoiceB === "rock" && playerChoiceA === "rock") {
+  console.log("It's a tie!")
+}
+else if(playerChoiceB === "paper" && playerChoiceA === "paper") {
+  console.log("It's a tie!")
+}
+else if(playerChoiceB === "scissors" && playerChoiceA === "scissors") {
+  console.log("It's a tie!")
+}
+
```

Comment:
@jl1982 Good job here using the conditional statements to check for the outcome of who will win the game. I would suggest keeping the indentation consistent inside your conditional statements and add one more conditional that would return an error if the input is incorrect.
Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L129)